NuguAgents/Interface/AudioPlayer/AudioPlayerState.swift

```diff
@@ -21,15 +21,33 @@
 import Foundation
 
 /// Identifies the player state.
-public enum AudioPlayerState: String {
+public enum AudioPlayerState {
     /// Initial state, prior to acting on the first Play directive.
-    case idle = "IDLE"
+    case idle
     /// Indicates that audio is currently playing.
-    case playing = "PLAYING"
+    case playing
     /// Indicates that audio playback was stopped due to an error or a directive which stops or replaces the current stream.
-    case stopped = "STOPPED"
+    case stopped
     /// Indicates that the audio stream has been paused.
-    case paused = "PAUSED"
+    case paused
```

Comment:
```suggestion
    case paused(temporary: Bool)
```
어차피 RawValue를 안쓰면 요렇게 가져가고 playerActivity에서 분기하는건 어떨까요-? 
내부적인 상태는 paused가 맞는데 분기가 복잡해질 것 같아서요.

Reply:
수정했습니다.